docs/Documentation/Development/Performance_Tools/ARM/map.md

```diff
@@ -0,0 +1,38 @@
+# How to run MAP
+
+## FastX Setup
+In order to run MAP you will need to have FastX. Follow instructions here to download the and install the desktop client and connect to DAV nodes using it: [FastX](https://nrel.github.io/HPC/Documentation/Software_Tools/FastX/fastx/)
+
+
+## Program Setup
+ARM-MAP can show you how much time was spent on each line of code. To see the source code in map, you must use a version of your code that is compiled with the debug flag. For most compilers, this is `-g`. Note: You should not just use a debug build but should keep optimization flags `-O0` turned on when profiling. 
+
+For more information, see the ARM-MAP documentation on getting started. In particular, if your program uses statically linked libraries, the map profiler libraries will not be automatically linked and you will need to do so yourself. 
```

Comment:
A link to the documentation referenced here would be helpful

Reply:
added a link